.github/workflows/test.yml

```diff
@@ -43,7 +43,7 @@ jobs:
         ARTIFACTORY_USERNAME: ${{ secrets.ARTIFACTORY_USERNAME }}
         ARTIFACTORY_API_KEY: ${{ secrets.ARTIFACTORY_API_KEY }}
       run: |
-        python -m pytest
+        python -m pytest --cov-config=.coveragerc.$RUNNER_OS
```

Comment:
@hagenw I want the configuration file to be extended to `".coveragerc.[Windows,macOS,Linux]"`. Do you know the correct syntax to do that?

Reply:
An alternative is that you use an `if` statement, e.g.

```
    - name: Test with pytest on Windows
      env:
        ARTIFACTORY_USERNAME: ${{ secrets.ARTIFACTORY_USERNAME }}
        ARTIFACTORY_API_KEY: ${{ secrets.ARTIFACTORY_API_KEY }}
      run: |
        python -m pytest --cov-config=.coveragerc.windows
      if: matrix.os == 'windows-latest'
```

Reply:
Thanks. I now used `runner.os` and looks like it worked.